chef_master/source/resource_windows_feature.rst

```diff
@@ -58,13 +58,18 @@ Properties
 ``feature_name``
    **Ruby Type:** Array, String | **Default Value:** ``'name'``
 
-   The name of the feature(s) or role(s) to install, if it differs from the resource block name.
+   The name of the feature/role(s) to install. The same feature may have different names depending on the underlying resource being used (ie DHCPServer vs DHCP; DNS-Server-Full-Role vs DNS).
```

Comment:
We'll want to get this description text actually added to the chef source code since it's generated from there.